client/src/components/player/AvailableClues.tsx

```diff
@@ -65,6 +67,10 @@ export const AvailableClues = () => {
         }
     };
 
+    if (takeOverClue) {
+        return <Redirect to={`/player/clue/${takeOverClue.tableOfContentId}`} />
+    }
```

Comment:
This is fine for now, but I wonder if this is something we could/should be handling at the router level